2023-12-20 16:43:07,056 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:43:07,056 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:44:00,459 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:44:00,459 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:44:29,096 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:44:29,096 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:45:26,619 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:45:26,626 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:15,046 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:15,046 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:37,461 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:37,461 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:39,376 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:46:39,376 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:49:11,586 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:49:11,596 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:51:58,586 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:51:58,586 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:15,596 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:15,596 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:37,641 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:37,641 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:41,436 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:41,436 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:41,996 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:41,996 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:42,346 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:52:42,346 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:53:06,256 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:53:06,256 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:53:49,846 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:53:49,846 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:55:03,233 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:55:03,234 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:55:04,746 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 16:55:04,746 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:01:02,765 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:01:02,765 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:01:05,535 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:01:05,535 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:10,378 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:10,379 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:14,170 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:14,171 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:52,731 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:07:52,731 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:00,277 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:00,277 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:42,026 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:42,036 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:44,086 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:09:44,101 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:07,166 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:07,166 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:20,576 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:20,576 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:22,076 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:22,076 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:43,787 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:10:43,787 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:21,126 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:21,126 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:22,447 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:22,447 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:25,376 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:25,376 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:37,316 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:37,316 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:51,467 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:51,467 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:53,067 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:11:53,067 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:18,951 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:18,951 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:21,237 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:21,237 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:31,996 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:12:33,146 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:15:12,537 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:15:12,538 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None error_FILE
2023-12-20 17:33:17,067 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:33:17,087 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:33:38,117 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:33:38,130 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:33:42,647 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:33:42,657 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:34:23,857 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:37:48,227 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:37:49,407 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:37:59,357 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:37:59,357 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:38:00,037 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:38:00,037 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:38:01,397 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:38:01,397 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:39:12,107 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:39:12,107 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:39:21,647 ERROR TEST выходит еррор сильно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-20 17:39:21,647 CRITICAL TEST выходит критикал конкретно C:\Users\N1\Desktop\sf06\sf06\news\views.py None
2023-12-22 15:13:48,185 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 8: 'get_available_languages', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000011BD8EAAF80>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'get_available_languages'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 8: 'get_available_languages', expected 'endblock'. Did you forget to register or load this tag?
2023-12-22 15:13:48,198 ERROR "GET /test HTTP/1.1" 500 187780 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:15:34,688 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000011BD8DF4200>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2023-12-22 15:15:34,703 ERROR "GET /test HTTP/1.1" 500 171239 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:15:35,859 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000011BD907A100>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2023-12-22 15:15:35,868 ERROR "GET /test HTTP/1.1" 500 171239 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:16:13,238 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000011D2DC15980>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2023-12-22 15:16:13,258 ERROR "GET /test HTTP/1.1" 500 171239 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:16:37,038 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000011D2DCE6300>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2023-12-22 15:16:37,089 ERROR "GET /test HTTP/1.1" 500 171239 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:19:42,597 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name."), <traceback object at 0x0000011D2DCEE900>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name.
2023-12-22 15:19:42,609 ERROR "GET /test HTTP/1.1" 500 171007 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:19:43,674 ERROR Internal Server Error: /test C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name."), <traceback object at 0x0000011D2DC437C0>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\sf06\news\views.py", line 96, in test
    return render(request, 'test.html', {'string':string})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name.
2023-12-22 15:19:43,684 ERROR "GET /test HTTP/1.1" 500 171007 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:45:56,548 ERROR Internal Server Error: /en/news C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 26: 'articles', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?"), <traceback object at 0x000001CC723EBC00>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'articles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\defaulttags.py", line 961, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 26: 'articles', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
2023-12-22 15:45:56,562 ERROR "GET /en/news HTTP/1.1" 500 230718 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
2023-12-22 15:45:58,208 ERROR Internal Server Error: /en/news C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 26: 'articles', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?"), <traceback object at 0x000001CC736B7A00>)
Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'articles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\defaulttags.py", line 961, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 26: 'articles', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
2023-12-22 15:45:58,218 ERROR "GET /en/news HTTP/1.1" 500 230718 C:\Users\N1\Desktop\sf06\venv\Lib\site-packages\django\core\servers\basehttp.py None
